packages/amplify-codegen-e2e-core/src/configure/index.ts

```diff
@@ -25,6 +25,21 @@ const defaultProjectSettings = {
   startCmd: '\r',
 };
 
+/**
+ * Supported regions:
```

Comment:
Can you add a note that the list must be in the same order as what is showing in `amplify configure`.